src/domdiv/main.py

```diff
@@ -1497,10 +1497,12 @@ def filter_sort_cards(cards, options):
 
     # Combine upgrade cards with their expansion
     if options.upgrade_with_expansion:
+        if not options.exclude_expansions:
+            options.exclude_expansions = []
```

Comment:
This is confusing, as it would assigned an empty list if it's already an empty list. Check should probably be `is None` instead of `not`.

Reply:
You are correct.  I will provide a correction.